core/modules/qproc/testQueryAna.h

```diff
@@ -108,6 +104,8 @@ std::string buildFirstParallelQuery(QuerySession& qs, bool withSubChunks=true) {
     return first.queries[0];
 }
 
+} // anonymous namespace
```

Comment:
I agree.  It was already there so I changed as little as possible.  I'll take a look at doing something better here.